src/chapters/sql-part1/joins.rst

```diff
@@ -0,0 +1,152 @@
+Joins
+=====
+
+.. index:: ! join
+
+A **join** combines two tables into one result set.
+We can use joins when we want to query two tables at the same time. 
+Whenever we join two tables, we have to specify the condition upon which the tables need to be joined.
+
+In SQL, there are four different types of joins:
+
+#. Inner Join
+#. Left Outer Join
+#. Right Outer Join
+#. Full Outer Join
+
+No matter which join you are working with, the general syntax for the query looks like so:
+
+.. sourcecode:: mysql
+   :linenos:
+
+   SELECT ColumnName1, ColumnName2, ....
+   FROM TableA
+   TYPEOFJOIN JOIN TableB ON TableA.ColumnName1 == TableB.ColumnName1;
+
+In this general query, we specified what columns we want (or we could have used the ``*`` to read data from all columns).
+We have also specified that ``TableA`` is the *left* table and that ``TableB`` is the *right* table.
+On line 3, we need to include the type of join as part of our query with the ``JOIN`` keyword and the condition upon which we are joining the tables.
+Our condition follows the ``ON`` keyword and tells MySQL what we believe to be matching records.
+This may mean we want to join on matching customer ids or matching dollar amounts or matching dates depending on the tables we are working with and what questions we need to answer.
+
+.. index:: ! inner join
+
+Inner Join
+----------
+
+Joining two tables on an **inner join** produces a result set that only includes the values that match in both tables.
+
+.. admonition:: Example
+
+   If we use an inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what guests are going to both the vow renewal and the wedding.
+
+   .. sourcecode:: mysql
+
+      SELECT LastName, FirstName
+      FROM JohnsonVowRenewal
+      INNER JOIN JohnsonWedding ON JohnsonVowRenewal.GuestID == JohnsonWedding.GuestID;
+
+   This query will give us a result set of the first and last names of the guests from the ``JohnsonVowRenewal`` table that are also in the ``JohnsonWedding`` table.
+
+   .. figure:: figures/innerjoin.png
+      :alt: Venn diagram highlighting just the center where the two circles meet.
+
+   The venn diagram above shows the result set highlighted in blue.
+
+.. index:: ! left outer join
+
+Left Outer Join
+---------------
+
+Joining two tables on a **left outer join** gives us a result set will include all values in the left table and any matching records from the right table.
+
+.. admonition:: Example 
+
+   If we use a left inner join to combine ``JohnsonWedding`` and ``JohnsonVowRenewal`` in a query, we can see what all of the guests that were invited to the wedding and any guests who were also invited to the vow renewal.
```

Comment:
"...left outer join..."

Also, for the second half of the sentence, how about, "...in a query, the result set includes _all_ of the guests invited to the wedding and any guests who were also invited to the vow renewal."

Reply:
Done!